libraries/Microsoft.Bot.Builder.Dialogs/Prompts/PromptValidatorContext.cs

```diff
@@ -23,5 +22,11 @@ internal PromptValidatorContext(ITurnContext turnContext, PromptRecognizerResult
         public PromptOptions Options { get; }
 
         public IDictionary<string, object> State { get; }
+
+        /// <summary>
+        /// Gets the number of times the prompt has been executed.
+        /// </summary>
+        /// <value>A number indicating how many times the prompt was invoked (starting at 1 for the first time it was invoked).</value>
```

Comment:
… and then if we change this back to 0 based this comment is wrong

Reply:
As discussed over the phone, I renamed the property to AttemptCount and left it starting at 1.